client/src/components/categorical/category/index.js

```diff
@@ -192,6 +194,20 @@ class Category extends React.Component {
       globals.categoryDisplayStringMaxLength
     );
 
+    if (
+      !schema?.annotations?.obsByName[metadataField]?.writable &&
```

Comment:
This component also receives `isUserAnno`, which is essentially the same concept & value as `.writable`.   Suggest that you only use one of these if they are the same concept & always have the same value.